website/templates/public/login.mako

```diff
@@ -60,6 +64,16 @@
             <div class="form-group">
                 <label for="inputEmail3" class="col-sm-3 control-label">Email</label>
                 <div class="col-sm-9">
+                    %if sign_up:
+                    <input
```

Comment:
Rather than duplicate this input, you should be able to do:

```mako
                    <input
                        type="email"
                        class="form-control"
                        data-bind="value: username"
                        name="username"
                        id="inputEmail3"
                        placeholder="Email"
                        ${'autofocus' if not sign_up else ''}
                    />
```